1-js/02-first-steps/06-type-conversions/article.md

```diff
@@ -144,7 +144,7 @@ The conversion follows the rules:
 |<code>true&nbsp;/&nbsp;false</code> | `1 / 0` |
 | `string` | The string is read "as is", whitespaces from both sides are ignored. An empty string becomes `0`. An error gives `NaN`. |
 
-**`ToBoolean`** -- occurs in logical operations, or can be performed with `Boolean(value)`.
+**`ToBoolean`** -- Occurs in logical operations, or can be performed with `Boolean(value)`.
```

Comment:
hi @iliakan. Thanks for your comment. I interpreted this section sort of like **a series of dictionary entries**: ToString, ToBoolean, etc. They are not technically sentences -- the clause that follows "ToBoolean" lacks a subject -- but more like **entry word/definition pairs**. Thus, in this scenario the dash isn't interrupting a sentence, but rather separating a dictionary entry word from its definition. The definition begins with "Occurs", so it is capitalized. 

But you are correct: if a dash interrupts a sentence, the word following the dash should not be capitalized. But that is not the situation here.

[link to dictionary entry diagram](http://image3.slideserve.com/6680063/parts-of-a-dictionary-entry-n.jpg)